tests/stress_tests/memcheck_tests/flags.h

```diff
@@ -24,6 +24,7 @@ static const char test_conf_message[] = "Optional. Path to a test config with de
 /// test_conf is an optional parameter
 DEFINE_string(test_conf, OS_PATH_JOIN({"stress_tests_configs", "memcheck_tests", "test_config.xml"}), test_conf_message);
 
+// TODO: env_conf key is deprecated
```

Comment:
I'm not aware of any user who would read deprecation warnings. Wouldn't it be simpler to just remove this option?

Reply:
It's a placeholder to remove the key. In this PR I removed use of key (but it still available via CLI), after merge I will remove it's appearance in product-configs and remove key from CLI at all. Removing in this PR will cause incompatibility in CI